docker.go

```diff
@@ -359,6 +409,51 @@ func (p *DockerProvider) daemonHost() (string, error) {
 	return p.hostCache, nil
 }
 
+// CreateNetwork returns the object representing a new network identified by its name
+func (p *DockerProvider) CreateNetwork(ctx context.Context, req NetworkRequest) (Network, error) {
```

Comment:
make sure you delete networks after the tests

Reply:
Addressed here: https://github.com/testcontainers/testcontainers-go/pull/76/commits/41baa6d602d6d8985acb216aa8468f88f07e4cc5

Reply:
you only addressed it in test, but the library should cleanup containers/networks/everything on exit

Reply:
I've used the Reaper approach to delete networks, as the `ryuk` container supports it